elastic/kibana
# [Fleet] add status telemetry to agents_per_version

## Summary

Closes https://github.com/elastic/ingest-dev/issues/1491

Querying agent statuses per version to add to telemetry.

How to test locally:
- enroll a few different version of agents
- wait up to 1h to see the telemetry task triggered
- check debug logs to see that the agents per version telemetry contains the status information

```
[2023-07-12T11:00:45.172+02:00][DEBUG][plugins.fleet] Agents per version telemetry: [{"version":"8.7.1","count":4,"healthy":4,"unhealthy":0,"updating":0,"inactive":0,"unenrolled":0,"offline":0},{"version":"8.2.1","count":3,"healthy":3,"unhealthy":0,"updating":0,"inactive":0,"unenrolled":0,"offline":0},{"version":"8.2.0","count":2,"healthy":0,"unhealthy":0,"updating":0,"inactive":0,"unenrolled":0,"offline":2}]
```


### Checklist
